### ISSUE TITLE
StCR not appearing after install

### ISSUE THREAD

**Issue by lindsayvansomeren** [stcr/subscribe-to-comments-reloaded#326]
Hello, 

StCR is not appearing at the bottom of my comments form ([link to a post](https://www.notoriousdebt.com/diversify-your-income-avoid-financial-extinction/)) and I have no idea why. There is text for "you can manage the subscriptions of this post," but not checkbox. 

I am not using Jetpack or anything, and I've cleared my cache after updating the settings (see attached photo). 

I'm not really a programmer, but if you lay it out in plain English I can probably fix it. 

Any ideas? Thanks! 
![stcr](https://cloud.githubusercontent.com/assets/25986768/23270951/e36109a4-f9b2-11e6-9376-e675ea3d02e5.JPG)

**Comment by Reedyseth** [comment]
Hi @lindsayvansomeren ,

I just checked your site and the issue is related to your theme styles. Your theme is hiding the labels, as a result the checkbox is not visible. You will have to customize the checkbox elements under the **comment Form** tab, just take all the subscription elements out of the html label.

I apologize for not explaining this in plain English, however, there is no other way to speak about the issue and the solution. If you can't solve this you can hire my **PRO** support and I will be glad to fix it for you.

Regards.!!

**Comment by lindsayvansomeren** [comment]
Thanks @Reedyseth 

I'm not sure what a "subscription element" is, but I tried deleting all of the code in the Custom HTML box under the Comment Form tab and that didn't work. I then pasted the code back in and saved it since that didn't solve the problem. 

Can you please tell me how much the pro support costs, and how to proceed with this if it's within my price range?

Thanks again for your quick answer!

**Comment by Reedyseth** [comment]
> Can you please tell me how much the pro support costs, and how to proceed with this if it's within my price range?

Actually is not that expensive, it cost $25.00 dollars for a Year and you will also get access to the PRO options once they are complete, in the mean time I will make sure that the plugin is working for you. 
**Note**: The support is related only to the plugin issues

**Comment by lindsayvansomeren** [comment]
What are the other pro options? I can't seem to find a link to a pro version anywhere...

Thanks!

**Comment by Reedyseth** [comment]
> What are the other pro options? I can't seem to find a link to a pro version anywhere...

As I mentioned above, I am working on that version, is not visible to the public yet, but it includes:

* Settings importing/exporting.
* Subscribers importing/exporting so that you can migrate to another site.
* New Post notifications to your subscribers. You can use a checkbox or a widget
* Notification queue deliver for a large amount of subscribers (This will prevent you site to hangout while the notifications are deliver).
* Improve Management Page layout.

For now this are the options that will be available.

**Comment by lindsayvansomeren** [comment]
Oh, I see! My apologies. 

Yes, this does sound like something I would be interested in. The pro features sound cool and I'd be excited to try them once available, but if you are able to provide support in the meantime for the price given, I'd definitely pay for it. 

Do you have a payment system in the meantime, or can I send you a payment through Paypal or something? Do you need me to set you up with a User account through my site? I will need an email address if that is the case...

Thanks again.

**Comment by Reedyseth** [comment]
> or can I send you a payment through Paypal

Yes Paypal will work, please send me an email to reedyseth [at] gmail {dot] com so that I can give you more details.
